server/templates/station.html

```diff
@@ -26,7 +26,14 @@ <h3>Ground Station  {{ station.name }}</h3>
         <b>About the station</b>: {{ station.descr }}
     </p>
     <p>
-        <b>Current configuration:</b> {{ station.config }}
+        <b>Current configuration:</b>
+        <ul>
+        {% for c in station.config %}
+        {% if station.config[c] %}
```

Comment:
Use the `is defined` statement. Now, the value that equals `0` or `""` will not be visible.

Reply:
Added as suggested.